nginx/conf.d/nginx.conf

```diff
@@ -1,31 +1,22 @@
-worker_processes 4;
-events { worker_connections 1024; }
```

Comment:
The `worker_processes` should be defined, as the [default is just `1`](http://nginx.org/en/docs/ngx_core_module.html#worker_processes).
We might want the `worker_connections` to also be higher than the [default `512`](http://nginx.org/en/docs/ngx_core_module.html#worker_connections).

Reply:
Yes, I need to look more into this as well. I had these initially but it didn't work out of the box with the nginx-ctrtbot image we are using. Should be no problem to append these lines later though.

Reply:
I see, I've added a reminder for this in MA-851.